index.html

```diff
@@ -2825,125 +2827,265 @@ <h3>Working with personal names</h3>
+	<details class="links"><summary>source</summary>
 	<p><a href="https://www.w3.org/International/questions/qa-personal-names#otherthings">Other things</a>, in <cite>Personal Names Around the World</cite>.</p>
 	</details>
+	<details class="links"><summary>examples</summary>
+	<p><a href="https://www.w3.org/International/questions/qa-personal-names#examples">Examples of differences</a> in <cite>Personal Names Around the World</cite>.</p>
+	</details>
 	</div>
 
+<p>This ensures that names are correctly handled for people such as Dina Asher-Smith and Christopher O'Connell.  Note that the apostrophe may appear as <span class="codepoint" translate="no"><bdi lang="en">&#x0027;</bdi> [<span class="uname">U+0027 APOSTROPHE</span>]</span> or as <span class="codepoint" translate="no"><bdi lang="en">&#x02BC;</bdi> [<span class="uname">U+02BC MODIFIER LETTER APOSTROPHE</span>]</span>, or perhaps even <span class="codepoint" translate="no"><bdi lang="en">&#x2019;</bdi> [<span class="uname">U+2019 RIGHT SINGLE QUOTATION MARK</span>]</span>.  A hyphen may be represented using <span class="codepoint" translate="no"><bdi lang="en">&#x002D;</bdi> [<span class="uname">U+002D HYPHEN-MINUS</span>]</span> or <span class="codepoint" translate="no"><bdi lang="en">&#x2010;</bdi> [<span class="uname">U+2010 HYPHEN</span>]</span>.</p>
+
 	<div class="req" id="loc_names_uppercase">
 	<p class="advisement">Don't require names to be entered all in upper case.</p>
-	<details class="links"><summary>explanations &amp; examples</summary>
+	<details class="links"><summary>source</summary>
 	<p><a href="https://www.w3.org/International/questions/qa-personal-names#otherthings">Other things</a>, in <cite>Personal Names Around the World</cite>.</p>
 	</details>
+	<details class="links"><summary>examples</summary>
+	<p><a href="https://www.w3.org/International/questions/qa-personal-names#examples">Examples of differences</a> in <cite>Personal Names Around the World</cite>.</p>
+	</details>
 	</div>
 
 	<div class="req" id="loc_names_case_normalization">
 	<p class="advisement">Don't normalize the casing in names.</p>
-	<details class="links"><summary>explanations &amp; examples</summary>
+	<details class="links"><summary>source</summary>
 	<p><a href="https://www.w3.org/International/questions/qa-personal-names#otherthings">Other things</a>, in <cite>Personal Names Around the World</cite>.</p>
 	</details>
+	<details class="links"><summary>examples</summary>
+	<p><a href="https://www.w3.org/International/questions/qa-personal-names#examples">Examples of differences</a> in <cite>Personal Names Around the World</cite>.</p>
+	</details>
 	</div>
 
+<p>Some names (such as 'McNamara') contain capital letters that are not the first letter; others (such as 'van der Waals') include words that are not capitalized. Forms should preserve the case the user enters and not coerce such names to always and only use capital letters at the start of each word.</p>
```

Comment:
Consider making `always and` be `always or`

Reply:
done